doc/examples/xx_applications/plot_morphology.py

```diff
@@ -12,12 +12,19 @@
 In this document we outline the following basic morphological operations:
 
 1. Erosion
+
 2. Dilation
+
 3. Opening
+
 4. Closing
+
 5. White Tophat
+
 6. Black Tophat
+
 7. Skeletonize
+
```

Comment:
it is not, thanks (it was me puzzled about other errors and trying things to solve them...)